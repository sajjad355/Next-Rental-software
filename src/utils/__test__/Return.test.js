import React from 'react';
import ReactDOM from 'react-dom';
import { render, cleanup } from '@testing-library/react';
import renderer from 'react-test-renderer';
import ReturnModal from '../../components/ReturnProduct/Return';
import '@testing-library/jest-dom/extend-expect';
import TestRenderer from 'react-test-renderer';


afterEach(cleanup);

it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(
        <ReturnModal></ReturnModal>, div
    );
});

// it('should match snapshot', () => {
//     const tree = renderer
//         .create(<ReturnModal />)
//         .toJSON();
//     expect(tree).toMatchSnapshot();
// });