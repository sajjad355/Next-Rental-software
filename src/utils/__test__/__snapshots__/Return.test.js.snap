// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<div
  className=""
>
  <div
    className="mb-5 book-return"
  >
    <button
      className="return btn btn-danger"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      Return
    </button>
  </div>
</div>
`;
